rustls/src/suites.rs

```diff
@@ -328,6 +332,10 @@ pub static TLS13_CHACHA20_POLY1305_SHA256: SupportedCipherSuite =
             aead_algorithm: &ring::aead::CHACHA20_POLY1305,
         },
         hkdf_algorithm: ring::hkdf::HKDF_SHA256,
+        #[cfg(feature = "quic")]
+        confidentiality_limit: u64::MAX,
+        #[cfg(feature = "quic")]
+        integrity_limit: 68_719_476_736, // MSRV 1.50: 2u64.pow(36)
```

Comment:
Added the literals here to be somewhat more conservative about the MSRV.

Reply:
You could use `integrity_limit: 1u64 << 36`

Reply:
Duh, I knew there was a better way I was missing.